docs/sdks/js-ts/overview.mdx

```diff
@@ -83,7 +83,7 @@ const dbid = Utils.generateDBID('account_identifier', 'database_name')
 
 ## Signers
 
-Certain operations in Kwil require signature authentication from the user (e.g. deploy database, drop database, execute CRUD actions, etc).
+Certain operations in Kwil require signature authentication from the user (e.g. deploy database, drop database, execute CUD actions, read actions in private mode, etc).
```

Comment:
Actions are more of a legacy feature, so we want to push procedures for now.

```suggestion
Certain operations in Kwil require signature authentication from the user (e.g. deploy database, drop database, execute CUD actions, read procedures in private mode, etc).
```

Reply:
This is not resolved.

Reply:
![Screenshot 2024-10-11 at 3 04 38 PM](https://github.com/user-attachments/assets/805d8f69-f374-43d5-ae86-b44e9d31c313)


@KwilLuke